pkg/apis/build/v1alpha2/build.go

```diff
@@ -141,6 +141,13 @@ func (b *Build) NotaryV1Config() *corev1alpha1.NotaryV1Config {
 	return b.Spec.Notary.V1
 }
 
+func (b *Build) DefaultProcess() string {
+	if b.Spec.DefaultProcess == "" {
+		return "web"
```

Comment:
I think we only want to fall back to web if we are using the `0.4` and `0.5` platform api. On `0.6` we should not set a default process type as that would override the default process type set by builpacks?